packages/block-library/src/latest-comments/index.php

```diff
@@ -42,7 +42,7 @@ function wp_latest_comments_draft_or_post_title( $post = 0 ) {
  *
  * @return string Returns the post content with latest comments added.
  */
-function render_block_core_latest_comments( $attributes = array() ) {
```

Comment:
Note: The `$attributes` variable is guaranteed to be an `array`, so the default value should be unnecessary.